Vindicta.Altis/UI/InGameMenu/InGameMenuTabGameModeInit.sqf

```diff
@@ -146,6 +146,8 @@ CLASS(__CLASS_NAME, "DialogTabBase")
 		pr _args = [clientOwner, _gameModeClassName, _gameModeParams, _campaignName, _templatesVerify];
 		CALLM2(gGameManagerServer, "postMethodAsync", "initCampaignServer", _args);
 
+		// Close in game menu after creating
+		CALLM0(gInGameMenu, "close");
```

Comment:
Can you remind me, why can't we just delete it instead? Why do we need a 'close' method? I've made the destructor of OOP DialogBase class close the dialog.

Reply:
I never made this class so I don't know.

Reply:
Oh wait, deleteOnNextFrame is actually my method, I forgot about that. I think I made it for a reason then. Probably after I crashed my game. Yes then it's cool!

Reply:
yeah this definitely crashed without using next frame delete.